pkg/daemon/config.go

```diff
@@ -17,12 +17,29 @@ import (
 	ptpv1 "github.com/openshift/ptp-operator/api/v1"
 )
 
+type nProfiles []ptpv1.PtpProfile
+
+// Len is required by sort.Interface
+func (p nProfiles) Len() int {
+	return len(p)
+}
+
+// Less is required by sort.Interface
+func (p nProfiles) Less(i, j int) bool {
+	return *p[i].Name < *p[j].Name
```

Comment:
This is being dropped from the sorting in master?
```
		aHasPhc2sysOpts := a.Phc2sysOpts != nil && *a.Phc2sysOpts != ""
		bHasPhc2sysOpts := b.Phc2sysOpts != nil && *b.Phc2sysOpts != ""
		//sorted in ascending order
		// here having phc2sysOptions is considered a high number
		if !aHasPhc2sysOpts && bHasPhc2sysOpts {
			return -1 //  a<b return -1
		} else if aHasPhc2sysOpts && !bHasPhc2sysOpts {
			return 1 //  a>b return
		}
```

Reply:
yes we are not bringing HA related changes to 4.15

Reply:
@josephdrichard  aHasPhc2sysOpts && bHasPhc2sysOpts is HA releated

Reply:
@josephdrichard sorry it is not HA related , It is generic. You are right  we should include that logic . @jzding